.travis.yml

```diff
@@ -4,28 +4,44 @@ sudo: false
 matrix:
     include:
         - python: "2.7"
-          env: TOXENV=docs,lint-py27
+          env: TOXENV=docs,lint
         - python: "3.6"
-          env: TOXENV=docs,lint-py36
+          env: TOXENV=docs,lint
+# Do not test Python 2.6 anymore, it is unsupported
+#        - python: "2.6"
+#          env: TOXENV=py26,py26-datetime
         - python: "2.7"
-          env: TOXENV=py27
-        - python: "2.7"
-          env: TOXENV=py27-datetime
+          env: TOXENV=py27,py27-datetime
+# Do not test Python 2.6 anymore, it is unsupported
+#        - python: "3.4"
+#          env: TOXENV=py34,py34-datetime
```

Comment:
I'd suggest to remove these commented lines. I see no gain to keep them, especially as the documentation comment is wrong for some of the versions.